### DISCUSSION TITLE
p/follow-app/

### DISCUSSION THREAD

**Discussion by giscus** [debuginn/blog#316] [General]
# p/follow-app/

近期体验了一款高效的私有化信息订阅平台，给大家推荐一下，官方则是描述为 Next-gen information browser，下一代的信息浏览器，它不单纯是一个 RSS 订阅软件，因为在传统的 RSS 仅仅订阅的是网站的 feed 流信息进行 follow。\n而 Follow 则是与 RSSHub、RSS3 相结合的生态软件，从而扩展了很多的订阅源以供订阅。\nFollow Follow 不单单提供了 RSS 的订阅，它的定位更是官网中所描述的 “下一代的信息浏览器”，目前它提供的功能有：\nArticles 文章订阅：偏向于传统的 RSS 订阅的模式； Social Media 社交媒体：更加偏向于 Follow 自己刚兴趣的作者在提供订阅的 APP 上； Pictures 图库订阅：订阅自己感兴趣的图片网站； Videos 视频订阅：订阅自己的在 Youtube、BiliBili 等视频平台的 UP 主； Audio 音频订阅：更像是播客类的订阅 APP； Notifications 通知订阅：订阅新闻媒体的信息； 详见官网地址：https://follow.is/ Github 地址：https://github.com/RSSNext/follow\n逆算法化 在如今信息化时代，大家接收到的信息都是经过大数据、算法进行点对点的信息推送，导致大家的信息来源越来越趋向于单一、被动接受、羊群效应，大家少了思考、多了短暂地愉悦。\n在 23 年，我就写了一篇文章叫做 《2023 年为何我还在使用 RSS》，文中我列举了 5 点好处，这也是我为什么大学毕业以来一直使用 RSS 阅读器的原因。\n其实大家动不动刷抖音、快手、小红书或者微信视频号等短视频平台就会发现，短则半个小时，长则 4-5 个小时的时间都浪费在了这些 APP 上面，回头来看，看过了什么视频，学到了啥，基本上就属于短暂地愉悦，长时间来看就像游戏一种性质。\n个人定制化的信息输入才是对自己认知与能力的快速提升。\n

https://blog.debuginn.com/p/follow-app/

**Comment by soctday** [comment]
v2ex过来。求邀请码。邮箱 soctday#gmail.com.感谢。

**Comment by yogurut** [comment]
v2过来的 yoguruto_@outlook.com

**Comment by Rlxzmdd** [comment]
v2ex过来的，求邀请码😘😘rlxzmdd@icloud.com

**Comment by HBUzxl** [comment]
订阅作者的rss链接看见的这篇文章，希望可以抽中 zxl2024@qq.com

**Comment by xi-mad** [comment]
v2ex过来。求邀请码。邮箱 1171866049@qq.com 感谢。

**Comment by Raul1718** [comment]
求邀请码 ～  zhangzuoqiang1018@gmail.com

**Comment by xiefenga** [comment]
v2ex过来的，求邀请码😘😘  1294699027@qq.com

**Comment by tang95** [comment]
关注很久了，v2过来的 153929962@qq.com

**Comment by debuginn** [comment]
恭喜 @Raul1718 获得邀请码，已经邮件联系了哈

**Comment by pangkuku** [comment]
大佬，v2过来的，求一个邀请码541985037@qq.com，感谢

**Comment by zhukai78** [comment]
大佬，v2ex过来的，求一个邀请码zl282145321@gmail.com，感谢

**Comment by debuginn** [comment]
大家不要在这里评论了哈，请公众号联系获取哈～
